src/Components/Navbar.js

```diff
@@ -1,22 +1,35 @@
-import React from 'react'
+import React, { useState } from 'react'
 import logo from '../assets/discify-logo.png'
+import DropdownMenu from './DropdownMenu'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faBars } from '@fortawesome/free-solid-svg-icons'
 import '../Styles/Navbar.scss'
 
-const Navbar = ({toggleClick}) => {
+const Navbar = () => {
+
+  const [clicked, setClick] = useState(false)
+
+  const toggleClick = () => {
+    console.log(clicked)
+    setClick(!clicked)
+  }
 
   return (
-    <nav>
+    <>
+      <nav>
       <img
         className='logo'
         src={logo}
         alt='Discify Logo'
       />
-      <button className='dropdown-button' onClick={() => toggleClick()}>
-        <FontAwesomeIcon icon={ faBars }/>
+      <button
```

Comment:
Thank you for adjusting this button component and attributes much easier to read